src/main/java/com/epam/container/TransportList.java

```diff
@@ -0,0 +1,328 @@
+package com.epam.container;
+
+import com.epam.transport.Transport;
+
+import java.util.*;
+import java.util.function.Predicate;
+
+/**
+ * Class TransportList that implements and contains methods of List interface.
+ */
+public class TransportList implements List<Transport> {
+
+    private final int NUMBER_OF_FIELDS = 4;
+    private final static int INITIAL_CAPACITY = 5;
+    private int size;
+    Transport[] array;
```

Comment:
1. array should be private? try to rename it to more appropriate name, like container or elements